packages/key-did-resolver/src/secp256r1.ts

```diff
@@ -0,0 +1,51 @@
+import * as u8a from 'uint8arrays'
+import multicodec from 'multicodec'
+import  multibase from'multibase'
+
+/**
+ * Constructs the document based on the method key
+ */
+export function keyToDidDoc (pubKeyBytes: Uint8Array, fingerprint: string): any {
+  const did = `did:key:${fingerprint}`
+  const keyId = `${did}#${fingerprint}`
+  const key = pubKeyBytesToXY(pubKeyBytes);
+  return {
+    id: did,
+    verificationMethod: [{
+      id: keyId,
+      type: 'JsonWebKey2020',
```

Comment:
What's the reasoning behind using `JsonWebKey2020` instead of `publicKeyBase58`?

Reply:
I will ask.

Reply:
See https://lists.w3.org/Archives/Public/public-credentials/2021Apr/0139.html .

Reply:
Ok, thanks for clarifying. I think this makes sense then.